test/Microsoft.Framework.PackageManager.FunctionalTests/DnuListTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Framework.FunctionalTestUtils;
+using Xunit;
+
+namespace Microsoft.Framework.PackageManager.FunctionalTests
+{
+    public class DnuListTests : IClassFixture<DnuListTestContext>, IDisposable
+    {
+        private readonly DnuListTestContext _context;
+        private readonly DisposableDir _workingDir;
+
+        public DnuListTests(DnuListTestContext context)
+        {
+            _context = context;
+        }
+
+        public static IEnumerable<object[]> RuntimeComponents
+        {
+            get { return TestUtils.GetRuntimeComponentsCombinations(); }
+        }
+
+        public void Dispose()
+        {
+            _workingDir.Dispose();
+        }
+
+        [Theory]
+        [MemberData("RuntimeComponents")]
+        public void DnuList_EmptyProject_Default(string flavor, string os, string architecture)
+        {
+            var runtimeHomePath = _context.GetRuntimeHome(flavor, os, architecture);
+
+            var projectJson = Path.Combine(_workingDir.DirPath, "project.json");
+            File.WriteAllText(projectJson, @"{}");
+
+            string stdOut, stdErr;
+            Assert.Equal(0, DnuTestUtils.ExecDnu(runtimeHomePath, "list", "", out stdOut, out stdErr, environment: null, workingDir: _workingDir.DirPath));
```

Comment:
Another way of looking at it is, if you make a minor output format change you should be checking the tests and updating them. Also, it can catch minor output bugs, I just found one yesterday because I was checking the output in some tests.